display.py

```diff
@@ -283,6 +283,7 @@ def run(self):
                 draw = ImageDraw.Draw(image)
                 draw.rectangle((0, 0, self.shared_data.width, self.shared_data.height), fill=255)
                 draw.text((int(37 * self.scale_factor_x), int(5 * self.scale_factor_y)), "BJORN", font=self.shared_data.font_viking, fill=0)
+                draw.text((int(110 * self.scale_factor_x), int(5 * self.scale_factor_y)), f"IP: {get_ip_address()}", font=self.shared_data.font_arial9, fill=0)
```

Comment:
It might be better to retrieve the IP address once and store it in an instance variable, and only update when necessary (during network changes)

```python3
# initialization or network change event
self.ip_address = get_ip_address()

# line 286
draw.text((int(110 * self.scale_factor_x), int(5 * self.scale_factor_y)),
          f"IP: {self.ip_address}", font=self.shared_data.font_arial9, fill=0)
```